templates/taxi/car_detail.html

```diff
@@ -1,6 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <a href="{% url 'taxi:car-update' pk=car.id %}" style="float: right">
+    <button type="button" class="btn btn-secondary btn-lg">
+     Update car
+    </button>
+  </a>
+  <a href="{% url 'taxi:car-delete' pk=car.id %}" style="float: right">
+    <button type="button" class="btn btn-danger btn-lg">
+     Delete car
+    </button>
+  </a>
+
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>Drivers</h1>
```

Comment:
To be honest, I don't like the look of this page. Mostly because the word `Drivers` is so big, even though it's not the main thing at all on this page. Changes tag `h` and make your page prettier